### ISSUE TITLE
valid_X为未定义的变量：

### ISSUE THREAD

**Issue by Cgetier520990** [aigc-apps/AMFormer#13]
在data_hcder.py中，在 __init__ 方法中，如果 dataset == 'valid' 时，代码中引用了 valid_X 和 valid_Y，但这些变量没有定义。

**Comment by Ch3ngY1** [comment]
可以通过修改以下这段内容：

**Comment by Ch3ngY1** [comment]
可以修改以下内容
        if dataset in ['train', 'test']: 
            train_X, test_X, train_Y, test_Y = train_test_split(data, labels, test_size=0.2, random_state=args.seed)
            if args.val_split:
                train_X, _, train_Y, _ = train_test_split(train_X, train_Y, test_size=0.185, random_state=args.seed)
来规定你的train和val数据的划分
